packages/experimental/RadioGroup/src/RadioGroup/RadioGroupTokens.ts

```diff
@@ -6,4 +6,7 @@ export const defaultRadioGroupTokens: TokenSettings<RadioGroupTokens, Theme> = (
   ({
     color: t.colors.menuItemText,
     variant: 'subtitle2Strong',
+    disabled: {
+      color: t.colors.neutralForegroundDisabled,
```

Comment:
Does it make sense for the overall label of the radio group to have disabled text coloring? I would have expected to just have all the options greyed out instead.

Reply:
Oh okay. I think that's weird, but if that's how web does it then sure.